test/contracts/upgrader/output/upgrader.imports.json

```diff
@@ -0,0 +1,7 @@
+[
+    "finish",
+    "getArgument",
+    "upgradeFromSourceContract",
+    "createContract",
+    "upgradeContract"
+]
```

Comment:
empty line

Reply:
We really have to fix this in `erdpy`...